go.mod

```diff
@@ -1,13 +1,19 @@
 go 1.18
 
+retract [v3.0.0, v3.3.0]
+
 module github.com/cosmos/ibc-go/v3
 
+// dragonberry replace for ics23
+replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
+
+// protocol buffers replace
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
```

Comment:
nit: should we group the `replace`s at the bottom:

```go
replace (
    // ics23
    // protocol buffers
)

Reply:
sure thing